hack/dind-cluster.sh

```diff
@@ -142,6 +213,34 @@ function stop() {
   sudo rm -rf "${config_root}"/openshift.local.etcd
   sudo rm -rf "${config_root}"/openshift.local.config
 
+  clean-orphaned-storage
+}
+
+function copy-image() {
+  local cluster_id=$1
+  local image=$2
+
+  echo "Installing image '${image}' on all nodes of dind cluster '${cluster_id}'"
+
+  # Error if a cluster is not configured
+  check-containers "copy-image"
+  check-no-containers "copy-image" "-f status=exited" "Paused parts"
+
+  # Make a temp file with a descriptor attached
+  tmpfile=$(mktemp /tmp/docker-image.XXXXXX)
+  exec 3>"$tmpfile"
+  rm "$tmpfile"
```

Comment:
The fd thing is not to avoid the temp file, and it's not for security (hopefully the mktemp covers that) it's just to make sure we don't leave an image laying around if we fail later.